tests/python/ExponentWithLinearTransformTest.py

```diff
@@ -0,0 +1,196 @@
+
+    def tearDown(self):
+        self.exp_tr = None
+
+    def test_direction(self):
+        """
+        Test the setDirection() and getDirection() methods.
+        """
+
+        # Default initialized direction is forward.
+        self.assertEqual(self.exp_tr.getDirection(),
+                         OCIO.TRANSFORM_DIR_FORWARD)
+
+        for direction in OCIO.TransformDirection.__members__.values():
+            # Setting the unknown direction preserves the current direction.
+            if direction != OCIO.TRANSFORM_DIR_UNKNOWN:
+                self.exp_tr.setDirection(direction)
+                self.assertEqual(self.exp_tr.getDirection(), direction)
+
+        # Wrong type tests.
+        for invalid in (None, 1, 'test'):
+            with self.assertRaises(TypeError):
+                self.exp_tr.setDirection(invalid)
+
+    def test_format_metadata(self):
+        """
+        Test the getFormatMetadata() method.
+        """
+
+        format_metadata = self.exp_tr.getFormatMetadata()
+        format_metadata.setName(self.TEST_ID)
+        self.assertIsInstance(format_metadata, OCIO.FormatMetadata)
+        self.assertEqual(format_metadata.getName(), self.TEST_ID)
+
+    def test_gamma(self):
+        """
+        Test the setGamma() and getGamma() methods.
+        """
+
+        # Default initialized vars value is [1, 1, 1, 1]
+        self.assertEqual(self.exp_tr.getGamma(), [1, 1, 1, 1])
+
+        self.exp_tr.setGamma(self.TEST_GAMMA)
+        self.assertEqual(self.exp_tr.getGamma(), self.TEST_GAMMA)
+
+        # Wrong type tests.
+        for invalid in (None, 'hello', [1, 2, 3]):
+            with self.assertRaises(TypeError):
+                self.exp_tr.setGamma(invalid)
+
+    def test_validate_gamma(self):
+        """
+        Test the validate() method.
+        """
+
+        # Validate should pass with default values.
+        self.exp_tr.validate()
+
+        # Validate should pass with positive values.
+        self.exp_tr.setGamma([1, 2, 3, 4])
+        self.exp_tr.validate()
+
+        # Validate should fail with lower bound value of 1.
+        self.exp_tr.setGamma([-1, -2, -3, -4])
+        with self.assertRaises(OCIO.Exception):
+            self.exp_tr.validate()
+
+        # Validate should fail with higher bound value of 10.
+        self.exp_tr.setGamma([11, 1, 1, 1])
+        with self.assertRaises(OCIO.Exception):
+            self.exp_tr.validate()
+
+    def test_negative_style(self):
+        """
+        Test the setNegativeStyle() and getNegativeStyle() methods.
+        """
+
+        # Default initialized negative style is clamp.
+        self.assertEqual(self.exp_tr.getNegativeStyle(), OCIO.NEGATIVE_LINEAR)
+
+        for negative_style in OCIO.NegativeStyle.__members__.values():
+            # These negative extrapolations are not valid for
+            # MonCurve exponent style.
+            ExceptionNegatives = [OCIO.NEGATIVE_CLAMP, OCIO.NEGATIVE_PASS_THRU]
```

Comment:
Recommend using lower case ``exception_negatives`` to avoid confusion as a type.

Reply:
Addressed in 2d14c6c.